File: js/index-6.js
let quadSum = 0;
let quadSumResult = 0;
let sumQuad = 0;

for (let index = 0; index <= 100; index++) {
    sumQuad += Math.pow(index, 2);
    quadSum += index;
    quadSumResult = Math.pow(quadSum, 2);
}

console.log(quadSumResult);
console.log(sumQuad);

console.log(quadSumResult - sumQuad);

File: lua/index-6.lua
quadSum = 0
quadSumResult = 0
sumQuad = 0

for i=1,100 do 
    sumQuad = sumQuad + i^2
    quadSum = quadSum + i
end

quadSumResult = quadSum^2
final = quadSumResult - sumQuad 


print (final)